app/js/app/controllers/AssignmentsController.js

```diff
@@ -228,11 +228,21 @@ define([], function() {
 		$scope.myAssignments.inProgressOld = [];
 
 		$scope.now = new Date();
-
 		var fourWeeksAgo = new Date($scope.now - (4 * 7 * 24 * 60 * 60 * 1000));
-		api.assignments.getMyAssignments().$promise.then(function(results) {
-			boardProcessor.augmentBoards(results);
-			angular.forEach(results, function(assignment, index) {
+
+		var extractBoardsFrom = function(assignments) {
+			var boards = []
+			for (var i = 0; i < assignments.length; i++) {
+				var assignment = assignments[i];
+				boards.push(assignment.gameboard);
+			}
+			return boards;
+		}
+
+		api.assignments.getMyAssignments().$promise.then(function(assignments) {
+			boardsForProcessing = extractBoardsFrom(assignments);
+			boardProcessor.augmentBoards(boardsForProcessing, null); // owner id null as the page uses assignerSummary
```

Comment:
Can we just use `$scope.user._id` instead of `null`?

Reply:
Yep, I agree that that is better.